docs/ballerina-by-example/examples/grpc-bidirectional-streaming/client/chat.sample.client.bal

```diff
@@ -0,0 +1,56 @@
+// This is client implementation for bidirectional streaming scenario
+package client;
+import ballerina/net.grpc;
+import ballerina/io;
+import ballerina/log;
+import ballerina.runtime;
+
+int total = 0;
+function main (string[] args) {
+
+     endpoint chatClient chatEp {
+            host: "localhost",
+            port: 9090
+        };
```

Comment:
Seems like the formatting is not correct.

Reply:
Fixed in f2b7cd26cb71721658fbc018e48d1aecaddaecd2